README.md

```diff
@@ -37,6 +39,8 @@ Deploy to your appengine instance OR run the AppEngineLauncher locally.
 
 1. If you see errors about missing the default Cloud Storage bucket, follow the [cloud integration instructions](https://cloud.google.com/appengine/docs/php/googlestorage/setup) to create a default bucket for your project.
 
+1. As a last resort, you can *clear the cache* by issuing a **GET** request to `https://YOUR_PROJECT_NAME.appspot.com/clear_cache.php`. You will be promoted for authentication, and then the script will delete symfony's cache from your Cloud Storage Bucket.
```

Comment:
it's almost a symfony inside joke that clearing the cache will fix your issues... a tremendous pain during development that has often been the case for bugs.

I will reword it, yes. Maybe something like "It may be useful to *clear the cache*. You can do this by..."